components/automate-ui/src/app/modules/desktop/desktop-detail/desktop-detail.component.html

```diff
@@ -0,0 +1,209 @@
+<div class="content-container">
+  <div class="header">
+    <div class="heading">
+      <div class="page-title">{{ desktop.name }}</div>
+      <!-- <div class="update-time">{{ desktop.name }}</div> -->
+    </div>
+    <div class="heading-actions">
+      <chef-button secondary (click)="fullscreenToggled.emit()">
+        <chef-icon *ngIf="!fullscreened">fullscreen</chef-icon>
+        <chef-icon *ngIf="fullscreened">fullscreen_exit</chef-icon>
+      </chef-button>
+      <chef-button secondary (click)="close()">
+        <chef-icon>close</chef-icon>
+      </chef-button>
+    </div>
+  </div>
+
+  <div class="checkin-history">
+    <div class="checkin-history-header">
+      <h3>Check-in History <span *ngIf="showCheckinDebug">(debug)</span></h3>
+      <div class="heading-actions">
+        <chef-button secondary (click)="showCheckinDebug = !showCheckinDebug">
+          <chef-icon>settings</chef-icon>
+        </chef-button>
+        <chef-button secondary (click)="checkinNumDays = (checkinNumDays === 14 ? 28 : 14)">
+          <span>Last {{ (checkinNumDays === 14 ? 2 : 4) }} Weeks</span>
+          <chef-icon>expand_more</chef-icon>
+        </chef-button>
+        <chef-button secondary>
+          <span>Download</span>
+          <chef-icon>expand_more</chef-icon>
+        </chef-button>
+      </div>
+    </div>
+    <div class="debug-actions" *ngIf="showCheckinDebug">
```

Comment:
Not sure if these are related
1. Theres some debug action going on here.  Good chance you left it here on purpose since development is ongoing.

2. There's also some console logging coming from desktop-detail.component.ts

<img width="437" alt="console logs" src="https://user-images.githubusercontent.com/16737484/81969221-71044f00-95d2-11ea-998d-4df288cd1dd6.png">

Reply:
Huh could've sworn I deleted that `.log`. But yea, I did want to leave in the rest of the bits for demo and debugging purposes while the final checkin graph area is still being worked on.